src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingResourceIntTest.kt

```diff
@@ -3364,6 +3364,1381 @@ class SentencingResourceIntTest : IntegrationTestBase() {
     }
   }
 
+  @Nested
```

Comment:
This test is a more less copy in spirit of existing test that copies all cases in a booking (which itself is not used and will be deleted in next PR)